From 0298ba0815750862b3cc08f9369c3879e4768ac1 Mon Sep 17 00:00:00 2001
From: Jan Engelhardt <jengelh@inai.de>
Date: Tue, 28 Oct 2014 18:45:19 +0100
Subject: [PATCH 15/62] profile-resetter: do not tick send-settings by default

When the user wishes to reset his preferences, Chromium would suggest
to send over the old settings to Google. I presume that this is for
Google to investigate into the setting combinations that led the user
to issue a rest.

Of course, we cannot let that happen by default. Google is not to get
any information.
---
 chrome/browser/profile_resetter/reset_report_uploader.cc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/chrome/browser/profile_resetter/reset_report_uploader.cc b/chrome/browser/profile_resetter/reset_report_uploader.cc
index fee0739a04ab8..c6581c384ceb0 100644
--- a/chrome/browser/profile_resetter/reset_report_uploader.cc
+++ b/chrome/browser/profile_resetter/reset_report_uploader.cc
@@ -20,7 +20,7 @@
 
 namespace {
 const char kResetReportUrl[] =
-    "https://sb-ssl.google.com/safebrowsing/clientreport/chrome-reset";
+    "trk:275:about:blank"; //https://sb-ssl.google.com/safebrowsing/clientreport/chrome-reset";
 
 GURL GetClientReportUrl(const std::string& report_url) {
   GURL url(report_url);
-- 
2.25.1

